apier/v1/apier.go

```diff
@@ -342,12 +331,6 @@ func (self *ApierV1) LoadTariffPlanFromStorDb(attrs AttrLoadTpFromStorDb, reply
 		}
 	}
 
-	if len(userKeys) != 0 && self.Users != nil {
```

Comment:
I think this one is not really needed since it is part of users but I guess it will do no harm since we also remove users in the near future